templates/patterns/gallery/gallery.ui_patterns.yml

```diff
@@ -109,3 +109,31 @@ gallery:
           meta: 'Copyright © 2021, Author and Licence'
           source: 'https://loremflickr.com/800/600/vilnius'
           media_type: 'image'
+    disable_overlay:
+      type: "boolean"
+      label: "Disable overlay"
+      description: "Disables the overlay functionality."
+      preview: true
+      escape: false
+    expandable:
+      type: "boolean"
+      label: "Expandable"
+      description: "Whether the 'See all' link is shown or all items are shown."
+      preview: true
+      escape: false
+    url:
```

Comment:
Is this how we handle links with patterns? `url` and `label` as separate fields? Shouldn't it be a single array field with label and url? I'm asking as I don't know well. For sure, `label` and `url` are not good names for something that is in fact the "Footer URL"

Reply:
Yes, that's how we use links because we need to pass a Link object to the ECL component so we build that in the pattern's template.

Reply:
And what if we need to have 2 links that do 2 different things? How do we handle that? In any case `url` and `label` are not good names. Should be prefixed with `footer_link_*` no?